manipulation/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
done https://github.com/ros-planning/moveit_ros/pull/721

could you create the PRs for setup_assistant, plugins, commander, etc? they will also need to be branched and have their default branches switched and then the rosinstall updated on moveit_docs ;-)